board.go

```diff
@@ -294,7 +305,7 @@ func PlaceSnakesRandomly(rand Rand, b *BoardState, snakeIDs []string) error {
 	}
 
 	for i := 0; i < len(b.Snakes); i++ {
-		unoccupiedPoints := GetEvenUnoccupiedPoints(b)
+		unoccupiedPoints := removeCenterCoord(b, GetEvenUnoccupiedPoints(b))
```

Comment:
This would cause issues with the desire to place food in the center of the map and error sometimes when a snake was randomly placed in the center.